heartcheck.gemspec

```diff
@@ -27,4 +27,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rack-test', '~> 0.6.0', '>= 0.6.3'
   # for documentation
   spec.add_development_dependency 'yard', '~> 0.8.0', '>= 0.8.7.6'
+  spec.add_development_dependency 'concurrent-ruby', '~> 0.9.1', '>= 0.9.1'
```

Comment:
Man there's some reason to add that dependency just to  `development`?

With that approach we need to add concurrent-ruby when use the gem in our projects.
```
gem 'heart-check'
gem 'concurrent-ruby'
```

WDYT?